test/tests/functional/pbs_multi_sched.py

```diff
@@ -2243,3 +2243,107 @@ def test_resv_from_job_in_multi_sched_using_rsub(self):
         a = {ATTR_job: jid, 'reserve_state': (MATCH_RE, 'RESV_RUNNING|5'),
              'partition': 'P1'}
         self.server.expect(RESV, a, id=rid)
+
+    def test_resv_alter_force_for_confirmed_resv(self):
+        """
+        Test that in a multi-sched setup when all schedulers are disabled
+        ralter -Wforce can still modify a reservation successfully even when
+        the ralter results into over subscription of resources.
+        """
+
+        self.common_setup()
+        # Submit 4 reservations to fill up the system and check they are
+        # confirmed
+        for _ in range(4):
+            t = int(time.time())
+            a = {'Resource_List.select': '1:ncpus=2', 'reserve_start': t + 300,
+                 'reserve_end': t + 900}
+            r = Reservation(TEST_USER, a)
+            rid = self.server.submit(r)
+            attr = {'reserve_state': (MATCH_RE, 'RESV_CONFIRMED|2')}
+            self.server.expect(RESV, attr, rid)
+            partition = self.server.status(RESV, 'partition', id=rid)
+            if (partition[0]['partition'] == 'P1'):
+                p1_start_time = a['reserve_start']
```

Comment:
Even if I do that, I cannot avoid status call because I need to get the reservation that's confirmed in partition P1. So while I have the status, it wouldn't be too bad to get it from the reservation :)

Reply:
Ahh OK, so you need to keep the status().  I'd still keep the reservation start and just use it.